### PR TITLE
Use conda-build 3

### LINKED ISSUE #4: @conda-forge-admin, please re-render
@conda-forge-admin, please re-render.

### PR DESCRIPTION
**Pull request by conda-forge-linter**
Hi! This is the friendly automated conda-forge-webservice.

I've rerendered the recipe as instructed in #4.

Here's a checklist to do before merging.
- [ ] Bump the build number if needed.

Fixes #4

### CODE REVIEW

**Comment by jakirkham** [comment]
@conda-forge-admin, please re-render.

**Comment by jakirkham** [comment]
@conda-forge-admin, please re-render.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -12,33 +12,21 @@ source:
...
-    - vc 9  # [win and py27]
-    - vc 10  # [win and py34]
-    - vc 14  # [win and py35]
 
 # Tests require visual context not present on CIs
 test:
   requires:
+    - {{ compiler("c") }}
     - cmake
-    - toolchain
```
Comment (jakirkham) [inline recipe/meta.yaml]: Not sure if this is the right way to require a compiler for a test. Could you please take a look, @msarahan?

File: .appveyor.yml [hunk 2]
```diff
@@ -4,39 +4,22 @@
...
-
-    - TARGET_ARCH: x64
-      CONDA_PY: 27
-      CONDA_INSTALL_LOCN: C:\\Miniconda-x64
-
-    - TARGET_ARCH: x86
-      CONDA_PY: 35
-      CONDA_INSTALL_LOCN: C:\\Miniconda35
+    - CONFIG: win_c_compilervs2008vc14
+      CONDA_INSTALL_LOCN: C:\Miniconda36-x64
```
Comment (jakirkham) [inline .appveyor.yml]: Hmm...this is wrong.

File: .appveyor.yml [hunk 3]
```diff
@@ -4,39 +4,22 @@
...
-      CONDA_PY: 36
-      CONDA_INSTALL_LOCN: C:\\Miniconda36
+    - CONFIG: win_c_compilervs2015vc14
+      CONDA_INSTALL_LOCN: C:\Miniconda36-x64
 
-    - TARGET_ARCH: x64
-      CONDA_PY: 36
-      CONDA_INSTALL_LOCN: C:\\Miniconda36-x64
+    - CONFIG: win_c_compilervs2015vc9
+      CONDA_INSTALL_LOCN: C:\Miniconda36-x64
```
Comment (jakirkham) [inline .appveyor.yml]: Same here.

**Comment by isuruf** [comment]
@conda-forge-admin, please re-render.

**Comment by jakirkham** [comment]
Thanks @isuruf. That looks much better. Do we know the cause?

Pushed some other commits, which appear to fix the CI builds and cleanup the recipe a bit.

**Comment by isuruf** [comment]
There were issues with conda-build, that were fixed recently

File: recipe/meta.yaml (hunk 1 above)
Comment (msarahan) [reply recipe/meta.yaml]: Yes, this is what I would recommend.

**Comment by dschreij** [comment]
Thanks guys. Sorry for being such a lousy maintainer. I hope to have some time for improvement soon! :)

**Pull request merged**